CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ For more information about changelogs, check
 [Keep a Changelog](http://keepachangelog.com) and
 [Vandamme](http://tech-angels.github.io/vandamme).
 
+## 0.26.3 - 2016-10-07
```

Comment:
@bruce-fs Tested locally and it works as expected.

Since you already added today's date here, can you also change `version.rb` and push-force here?

Then I can merge and release. Thanks!